pallets/utxo/README.md

```diff
@@ -15,8 +15,7 @@ To run the test cases, just run command `cargo test`.
     "_enum": {
       "Pubkey": "Pubkey",
       "CreatePP": "DestinationCreatePP",
-      "CallPP": "DestinationCallPP",
-      "ScriptHash": "H256"
```

Comment:
`ScriptHash` is still present in [code](https://github.com/mintlayer/core/blob/staging/pallets/utxo/src/lib.rs#L211), is this change correct?